.gitmodules

```diff
@@ -7,3 +7,6 @@
 [submodule "tests/litani"]
 	path = tests/litani
 	url = https://github.com/awslabs/aws-build-accumulator
+[submodule "docs/doxygen/doxygen-css"]
+	path = docs/doxygen/doxygen-css
+	url = https://github.com/jothepro/doxygen-awesome-css.git
```

Comment:
This looks nice, but a submodule seems pretty heavy-duty just for CSS.

Is there another way we can pull in the CSS?

Reply:
```
curl https://raw.githubusercontent.com/jothepro/doxygen-awesome-css/main/doxygen-awesome.css
```

Reply:
I agree the submodule felt pretty heavy for this. I think the benefits are fairly decent because it allows
1. Clear separation of code ownership
2. Ability to pin CSS versions
3. Easier management 

I'll push a commit that uses Curl to bring the CSS in but IMO I think this would lead to some discoverability issues / make it harder to develop locally.

Reply:
1. Do submodules indicate anything about ownership? We might or might not own the code in a submodule.
2. You can also do that with curl :) `curl https://raw.githubusercontent.com/jothepro/doxygen-awesome-css/bf51192c4c9b17202a096fa7c2657f2d5fc30da3/doxygen-awesome.css`
3. Strong disagree. Updating submodules is a nightmare. If contributors aren't careful, later commits actually undo updates.